imap_processing/tests/glows/test_glows_decom.py

```diff
@@ -64,51 +63,52 @@ def test_bad_header():
 
 
 def test_header(decom_test_data):
-    expected_hist = CcsdsData(
```

Comment:
We can fix this by adding a type check in `CcsdsData.__init__()` before it tries to retrieve the `raw_value`. Or, we might be able to update `CcsdsData` to remove the `raw_value` step altogether and then pass in a dict here? Again, I'm not clear when a `_Parameter` type is returned that needs to be opened up, vs just returning a direct python type (ie `int`)